### PR TITLE
Generalized Sphinx code to display inputs and outputs across Examples.

### LINKED ISSUE #1851: In Sphinx docs, create run_example to generically run examples.
The existing `gaussian_utils.py` file should be generalized. There should be a method `run_example` which takes a module and function as string(s), imports then and then executes with the remaining arguments of `run_example`.

As a second example, use hello world to also call this method.

### PR DESCRIPTION
**Pull request by mbopfNIH**
closes #1851

### CODE REVIEW

**Comment by mbopfNIH** [comment]
This doesn't include **kwargs inputs, but it does include a dict for the return values. This was necessary for it to work with both HelloWorld and SimpleGaussian. It can still use a little tweaking, but it "works" and should be somewhat extensible.

File: docs/source/example_utils.py [hunk 1]
```diff
@@ -21,13 +20,14 @@ def plot_image(image: SimpleITK.Image):
     plt.imshow(nda, cmap='gray')
 
 
-def run_gaussian():
+def run_example(module_name, func_name, args):
+    sys.path.append(f'../../Examples/' + module_name)
```
Comment (blowekamp) [inline docs/source/example_utils.py:24]: Please document this functions and arguments with doc strings: https://peps.python.org/pep-0257/

**Review by zivy — APPROVED** [review]

File: docs/source/example_utils.py (hunk 1 above)
Comment (blowekamp) [reply docs/source/example_utils.py:24]: You might as well use a standard as oppose to something arbitrary.

**Comment by mbopfNIH** [comment]
On the *args, **kwargs change, this will break any existing code, including any tests, that depend on the current input method. Not sure how much that matters, but I'll need to figure out how to update the tests.

**Comment by blowekamp** [comment]
The changes needed for args kwargs can be done in a separate PR.

File: docs/source/example_utils.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import SimpleITK
+import matplotlib.pyplot as plt
+import importlib
+
+# Hard-coding a screen dpi of 96: calculating it appears to be difficult
+SCREEN_DPI = 96
+
+
+def plot_image(image: SimpleITK.Image, cmap='gray'):
+    """This takes a SimpleITK Image and plots it using Maptplotlib.
```
Comment (blowekamp) [inline docs/source/example_utils.py:15]: sp:  Maptplotlib->Matplotlib

**Review by blowekamp — COMMENTED** [review]
LGTM

**Pull request merged**